molo/core/management/commands/switch_main_language.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import BaseCommand
+from molo.core.models import (
+    Languages, Main, SiteLanguage, SiteLanguageRelation, LanguageRelation)
+
+# This is currently made to work for single sites only
```

Comment:
Could we turn this into code rather than having a comment?
```
if WagtailSite.objects.count() > 1:
    fail(), exit(), error()
```

Reply:
sure